app/views/checkouts/address.html.haml

```diff
@@ -0,0 +1,93 @@
+= content_for(:title, t('order.checkout') + ' | ')
```

Comment:
Can we make it simpler?
I would like to read something like 
`= content_for(:title, t('order.checkout_title'))`
or just 
`= content_for(:title, title_for('checkout'))`, where `title_for` is helper